setup.py

```diff
@@ -249,8 +249,9 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'oauthlib!=2.0.3,!=2.0.4,!=2.0.5,<3.0.0,>=1.1.2',
     'requests-oauthlib==1.1.0',
 ]
-gcp = [
+google = [
     'PyOpenSSL',
+    'apache-beam',
```

Comment:
Oh. I'm afraid of that. Apache Beam is a very large project and has a lot of dependencies. In my opinion, this is not a library, but it is an application that the user must install separately. I and @tanjinP  are working on a guide that will describe it in more detail.
https://github.com/apache/airflow/pull/8809

After that, Apache Beam for Dataflow should be installed via `apache_beam[gcp]`.

Reply:
In other words. The user should always consciously install a specific version.

Reply:
Ok. I separated it to a separate extra.